sled-agent/src/params.rs

```diff
@@ -22,9 +22,18 @@ pub struct NetworkInterface {
     pub mac: external::MacAddr,
     pub subnet: external::IpNet,
     pub vni: external::Vni,
+    pub primary: bool,
     pub slot: u8,
 }
 
+/// An external IP address used for external connectivity for an instance.
+#[derive(Debug, Clone, Copy, Deserialize, Serialize, JsonSchema)]
+pub struct ExternalIp {
+    pub ip: IpAddr,
+    pub first_port: u16,
+    pub last_port: u16,
```

Comment:
Can we keep propagating the inclusivity/exclusivity comments here? Just wanna check everywhere is labelled, because it's subtle

Reply:
Yessir, will add.